litho-widget/src/main/java/com/facebook/litho/widget/RecyclerBinder.java

```diff
@@ -1422,11 +1423,20 @@ public void onScrolled(RecyclerView recyclerView, int dx, int dy) {
   private static class BaseViewHolder extends RecyclerView.ViewHolder {
 
     private final boolean isLithoViewType;
+    private ViewBinder viewBinder;
 
     public BaseViewHolder(View view, boolean isLithoViewType) {
       super(view);
       this.isLithoViewType = isLithoViewType;
     }
+
+    public ViewBinder getViewBinder() {
```

Comment:
I don't think you need getter/setter here as it is `private` inner class and you can access field directly.

Reply:
done